src/components/OrderDetails/OrderDetails.tsx

```diff
@@ -18,17 +17,11 @@ export const OrderDetails = ({ order }: Props): ReactElement => {
         <span className="order_details__label">Status:</span>
         <span className="order_details__value">{order.status}</span>
       </div>
-      <div className="order_details__group">
-        <span className="order_details__label">Address:</span>
-        <div className="order_details__value">
-          <FormattedAddress address={order.address} />
-        </div>
-      </div>
       <div className="order_details__group order_details__group--items">
         <span className="order_details__label">Items:</span>
         <div id="order_details__items">
-          {order.items.length === 0 ? <p>There are no items in your order.</p> : null}
-          {order.items.map((item) => (
+          {order.items?.length ? <p>There are no items in your order.</p> : null}
```

Comment:
I think this is wrong...

```tsx
// if length is truthy then it will display `here are no items in your order.` otherwise it will render `null`.
{order.items?.length ? <p>There are no items in your order.</p> : null}
```
I think the point of this logic was to only render that message if the length was 0 (or items is null?).

Reply:
You're right, I've just gotten the logic wrong. I changed it to `!order.items?.length`